pkg/apis/elasticsearch/v1/name.go

```diff
@@ -19,6 +19,7 @@ const (
 	configSecretSuffix                           = "config"
 	secureSettingsSecretSuffix                   = "secure-settings"
 	fileSettingsSecretSuffix                     = "file-settings"
+	esConfigSecretSuffix                         = "elasticsearch-config"
```

Comment:
```
k get secret | grep config                                         
elasticsearch-sample-es-default-es-config            Opaque   1      91m
elasticsearch-sample-es-elasticsearch-config         Opaque   2      91m
```

It's a bit difficult to understand what "config" is inside a `Secret` by reading its name, could use a prefix to make it clear when it's about a policy?